runtime/help/tutorial.md

```diff
@@ -1,33 +1,66 @@
 # Tutorial
 
-This is a brief intro to micro's configuration system that will give some
-simple examples showing how to configure settings, rebind keys, and use
-`init.lua` to configure micro to your liking.
+This is a brief intro to `micro` workflow with simple examples how to configure
+settings, keys, and use `init.lua`.
 
 Hopefully you'll find this useful.
 
-See `> help defaultkeys` for a list an explanation of the default keybindings.
+## Command mode
 
-### Settings
+Press `Ctrl-e` to open micro's command prompt. Typing `help tutorial` will open
+this documentation.
+
+For the rest of the docs `> help tutoral` indicates pressing `Ctrl-e`.
+
+## Default keyboard shortcuts
+
+Enter `> help defaultkeys` for a list of the default keybindings.
+
+## Simple workflow: Move content from one file to another
+
+For this example, we edit micro's own codebase, and move contents between files.
+
+Press `Ctrl-o` and enter filename to open the file in the current view
+(use `Tab` key to help with autocompletion)
+```
+> open internal/config/plugin_manager.go
+```
+Then from the command prompt (`Ctrl-e`) open a second file in a vertical split
+```
+> vsplit internal/config/plugin.go
+```
```

Comment:
No. For me `tab` is a key. =)

In `i3` I don't need tabs. The edit workflow when you can see two or more files in one window seems more useful to me. Maybe the real reason why I decided to use `micro` is because copying text between `vim` instances running in different tabs didn't work for me, or the contents was lost when `vim` exits.

Reply:
It's the opposite for me – I mainly use tabs because I keep my terminal window small and there's not really enough space for two buffers to fit comfortably. From my perspective tabs are the more basic form of operation (if you open multiple files at once the default behavior is to open them in tabs) and splits are slightly more advanced feature that users can learn about on their own. But I guess it ultimately doesn't really matter which one the tutorial uses.